src/app/ApplicationTemplate.Tests/Presentation/NavigationShould.cs

```diff
@@ -9,37 +9,37 @@ public partial class NavigationShould : NavigationTestsBase
 		[Fact]
 		public async Task NavigateEverywhere()
 		{
-			await AssertNavigateFromTo<OnboardingPageViewModel, WelcomePageViewModel>(p => p.NavigateToWelcomePage);
+			//await AssertNavigateFromTo<OnboardingPageViewModel, WelcomePageViewModel>(p => p.NavigateToWelcomePage);
```

Comment:
I don't understand why this test would be disabled. Can't we just adapt it?

Reply:
Adapting the tests is not part of the current scope, that's why I'm disabling them temporarily.
For instance, p.NavigateToWelcomePage, p.NavigateToCreateAccountPage, p.NavigateToLoginPage, p.NavigateToHomePage don't exist anymore.
The whole test needs to be reworked

Reply:
>Adapting the tests is not part of the current scope, that's why I'm disabling them temporarily.

@MatFillion I feel like we should add a checkbox in the PR template about adjusting automated tests. I feel like tests should evolve at the same time as any other changes.

Reply:
Yeah they should evolve but the whole app layout changed with this as we are replacing whole portions.

Reply:
I spoke with @carlh98  and him and @arianeleonard will restore/adapt the ones that still make sense. (like this "navigateeverywhere" one.